### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by becatz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +0,0 @@
-hi
-nnn
```
Comment (becatz) [inline README.md:2]: @staceb would you mind taking a look at this

Comment (becatz) [inline README.md:2]: and this one too please

Comment (staceb) [reply README.md]: Writing a reply

Comment (staceb) [reply README.md]: Writing a reply

Comment (staceb) [reply README.md]: writing a response

**Pull request closed without merging**